src/compile-3.1.env

```diff
@@ -1,7 +1,8 @@
 #!/bin/bash
 
 # shellcheck disable=1090
```

Comment:
These files in `src` are actually generated by the pipelines that auto bump the ruby versions and the templates are found in here: https://github.com/bosh-packages/ruby-release/tree/master/ci/templates/src

We can't remove the `BOSH_PACKAGES_DIR` because the bosh-cli [uses that](https://github.com/cloudfoundry/bosh-cli/blob/1e2a34e4d8e23c0116c0f8eeaae647e9a7f3e223/installation/pkg/compiler.go#L102) to provide an alternative to `/var/vcap/packages` when it has to compile releases locally when doing a `bosh create-env` command.

We just cut a new version of the bosh cli and we're going to take a crack at updating your changes to address the above issues so we can try re-vendoring the ruby-release into the bosh director today